triggers/daml/BUILD.bazel

```diff
@@ -22,6 +22,7 @@ version: 0.0.1
 dependencies:
   - daml-stdlib
   - daml-prim
+build-options: ["--ghc-option", "-Werror"]
```

Comment:
As discussed in Slack, this currently doesn’t work properly since `build-options`  does not respect `--project-root`. I’ll fix this separately.